client/src/App.tsx

```diff
@@ -26,7 +26,7 @@ function App(): JSX.Element {
               <Route path="/" component={NavBar} />
               <Route exact path="/login" component={Login} />
               <Route exact path="/signup" component={Signup} />
-              <ProtectedRoute exact path="/fileUpload" component={DesignSubmit} />
+              <Route exact path="/fileUpload" component={DesignSubmit} />
```

Comment:
Please use hyphenated case for route names with multiple words

Reply:
changed to ProtectedRoute, and also did some change in ProtectedRoute component.